.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Let's remove this on the next assignment the same way we did with node modules (you may need to add it on your .gitignore too).